plugin/types.go

```diff
@@ -22,10 +22,11 @@ type Repo struct {
 // System provides important information about the Drone
 // server to the plugin.
 type System struct {
-	Version string   `json:"version"`
-	Link    string   `json:"link_url"`
-	Plugins []string `json:"plugins"`
-	Globals []string `json:"globals"`
+	Version         string   `json:"version"`
+	Link            string   `json:"link_url"`
+	Plugins         []string `json:"plugins"`
+	Globals         []string `json:"globals"`
+	EscalatePlugins []string `json:"escalate_plugins"`
```

Comment:
Should this change also be present in https://github.com/drone/drone-go/blob/master/drone/types.go#L128 ?